buildme.pl

```diff
@@ -25,21 +25,34 @@
 my $windowsPerlDir = "C:\\perl";
 my $windowsPerlPath = "$windowsPerlDir\\bin\\perl.exe";
 
+my $icuBigEnd = "icudt[0-9][0-9]b.dat ";
+my $icuLittleEnd = "icudt[0-9][0-9]l.dat ";
+my $freebsdDirs = "i386-freebsd-64int ";
+my $solarisDirs = "i86pc-solaris-thread-multi-64int ";
+my $armLinuxDirs = "arm-linux armhf-linux ";
+my $sparcLinuxDirs = "sparc-linux sparc-unknown-linux ";
+my $mainLinuxDirs = "i386-linux x86_64-linux "
+my $linuxDirs = $armLinuxDirs . $sparcLinuxDirs . $mainLinuxDirs . "powerpc-linux ";
```

Comment:
If already you move everything to variables, why didn't you do the same for powerpc-linux?

Reply:
Probably a good point - but my initial goal was to compress the information by combining frequently coupled data, so it was easier to read and maintain. Now I see that if there is a sudden resurgence of PowerPC's that use Perl with 64int or thread or something, we ought to be flexible. Will fix.